src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/azuredevops/model/CommentPosition.java

```diff
@@ -0,0 +1,29 @@
+package com.github.mc1arke.sonarqube.plugin.ce.pullrequest.azuredevops.model;
+
+import java.io.Serializable;
+
+public class CommentPosition implements Serializable {
+    private Integer line;
```

Comment:
`int` rather than `Integer`, or is this `null`alble?

Reply:
Resolve